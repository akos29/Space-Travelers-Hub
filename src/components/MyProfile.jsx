/* eslint-disable linebreak-style */
import React from 'react';
import JoinedMissions from './mission/JoinedMissions';
import styles from './MyProfile.module.css';
import MyRocket from './rockets/MyRocket';

export default function MyProfile() {
  return (
    <>
      <h2>MyProfile</h2>
      <div className={styles.myProfile}>
        <div className={styles.myMissions}>
          <h3>My Missions</h3>
          <ul>
            <JoinedMissions />
          </ul>
        </div>
        <div className={styles.myRockets}>
          <h3>My Rockets</h3>
          <ul>
            <MyRocket />
          </ul>
        </div>
      </div>
    </>
  );
}
