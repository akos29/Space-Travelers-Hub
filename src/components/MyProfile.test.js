import React from 'react';
import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Store from '../redux/store';
import MyProfile from './MyProfile';

it('Rockets Component snapshot test', () => {
  const rockets = TestRenderer
    .create(
      <Provider store={Store}>
        <MyProfile />
      </Provider>,
    )
    .toJSON();
  expect(rockets).toMatchSnapshot();
});
