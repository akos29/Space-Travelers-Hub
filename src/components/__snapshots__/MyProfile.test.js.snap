// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rockets Component snapshot test 1`] = `
Array [
  <h2>
    MyProfile
  </h2>,
  <div
    className="myProfile"
  >
    <div
      className="myMissions"
    >
      <h3>
        My Missions
      </h3>
      <ul>
        You haven't joined any missions yet
      </ul>
    </div>
    <div
      className="myRockets"
    >
      <h3>
        My Rockets
      </h3>
      <ul>
        You haven't booked any rocket yet
      </ul>
    </div>
  </div>,
]
`;
